tests/rptest/tests/scaling_up_test.py

```diff
@@ -156,7 +156,7 @@ def create_topics(self, rf, partition_count):
 
     @property
     def producer_throughput(self):
-        return 5 * (1024 * 1024) if not self.debug_mode else 1000
+        return (1024 * 1024) if not self.debug_mode else 1000
```

Comment:
Any idea how fast the producer actually goes in CI? It did seem to me that it wasn't hitting this limit. Maybe in poll_status we can output the cumulative MB/s or smth.